waiter/bin/ci/run-integration-tests.sh

```diff
@@ -40,20 +43,27 @@ if [ ${MINIMESOS_EXIT_CODE} -ne 0 ]; then
     echo "minimesos failed to startup -- exiting"
     exit ${MINIMESOS_EXIT_CODE}
 fi
+$(minimesos info | grep MINIMESOS)
 
 # Start waiter
-${WAITER_DIR}/bin/run-using-minimesos.sh 9091 &
+WAITER_PORT=9091
```

Comment:
I would vote no. This script is meant to be invoked by travis, not by developers